how-to/customize-workspace/docs/how-to-define-endpoints.md

```diff
@@ -2,3 +2,71 @@
 > OpenFin Workspace is currently **only supported on Windows** although you can run the sample on a Mac for development purposes.
 
 [<- Back to Table Of Contents](../README.md)
+
+# How To Define Endpoints
+
+Endpoints are defined via settings and the endpointProvider:
```

Comment:
How to define endpoints needs an intro of what is an endpoint, or a link to another doc explaining